contracts/pool/pools/liquid/LiquidTokenLockingPoolBase.sol

```diff
@@ -3,16 +3,16 @@ pragma solidity 0.8.18;
 
 import "../TokenLockingPoolBase.sol";
 
-error UnlockPeriodNotFinished();
-error UnlockPeriodFinished();
-error UnlockPeriodExists();
-
 /**
  * @title LiquidTokenLockingPoolBase
  * @dev Provides functionality for locking and withdrawing tokens.
  * Tokens can be withdrawn by respecting the unlock period or immediately by paying a removal fee.
  */
 abstract contract LiquidTokenLockingPoolBase is TokenLockingPoolBase {
+    error UnlockPeriodNotFinished();
+    error UnlockPeriodFinished();
+    error UnlockPeriodExists();
```

Comment:
Mozes li prebacit ove errore ispod eventa, cisto da postujemo [solidity style guide](https://docs.soliditylang.org/en/latest/style-guide.html#order-of-layout)